omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskServiceWrapper.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.scheduler.TaskDefinition;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.api.context.Context;
+
+import java.util.Collection;
+
+public class TaskServiceWrapper {
+	
+	public TaskDefinition getTaskById(String id) {
+		return Context.getSchedulerService().getTask(Integer.parseInt(id));
```

Comment:
Shouldn't this be the task uuid?

Reply:
here, It hasn't that much of function. I used taskName and TaskId instead of that

Reply:
Do you know anywhere in our REST calls where clients pass integer id instead of uuid?

Reply:
Nope,

but the problem is, It hasn't the function to getTask by UUID here. So Shall I want to implement that also?

Reply:
That would mean modifying the core platform. In that case, just live it to get by name and integer id.

Reply:
Okay, So I will just go through the name and id by now